types/yup/lib/customLocale.d.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  MixedSchema,
```

Comment:
Why all the unused imports? `LocaleObject` is the only type used in the file, so the other imports shouldn't be needed here.

Reply:
I defined `LocaleObject` in this file at first, and forgot to remove them after moving it.
This is fixed, thanks for catching this, I should have been more careful.